fdbserver/WorkerInterface.actor.h

```diff
@@ -52,7 +52,7 @@ namespace ptxn {
 
 struct TLogGroup {
 	TLogGroupID logGroupId;
-	std::unordered_map<StorageTeamID, std::vector<Tag>> storageTeams;
+	std::map<StorageTeamID, std::vector<Tag>> storageTeams;
```

Comment:
Why changing it to ordered map?

Reply:
unordered_map seems to not have `serialize` method, and I need to serialize this data structure.

Reply:
To make `unordered_map` work with serialization, you can take a look at [how `map` is implemented](https://github.com/jzhou77/foundationdb/blob/d62b6075fd192bfb052cf8d3267b36cd35ee7870/flow/serialize.h#L250-L269) and add similar code for `unordered_map`.